src/components/auth/RegisterParent.vue

```diff
@@ -74,6 +77,7 @@
                 show-icon="pi pi-eye-slash"
                 hide-icon="pi pi-eye"
                 :feedback="false"
+                data-cy="parent-password"
```

Comment:
```suggestion
                data-cy="span-parent-password"
```
Can we be a bit more descriptive with these parent signup selectors? I usually start with the type of element (span, label, button, etc...). Sort of a block-element__modifier format.

Reply:
sure!